src/webview/cluster/app/clusterView.tsx

```diff
@@ -209,7 +209,7 @@ export default function addClusterView(props: ClusterViewProps) {
   };
 
   const handleDisabled = () => {
-    return (activeStep === 1 && fileName === '')
+    return ((activeStep === 0 || activeStep === 1) && fileName === '')
         || (activeStep === 2 && pullSecretPath === '');
```

Comment:
With this PR,  I can skip over selecting the the pull secret file. I think you need to update the check on line `213` to check `activeStep === 1` instead of 2.